frontend/src/components/common/table/table.tsx

```diff
@@ -0,0 +1,63 @@
+import { useTable } from 'react-table';
+import { ReactTableRenderEntitiesNames } from 'components/common/table/common/react-table-render-entities-names';
+import styles from './table/styles.module.scss';
+import { Column } from 'react-table';
+import { FC } from 'common/types/types';
```

Comment:
Please pay attention on imports order - dependencies should be imported as they go on code base (
eg
1. <imports from packages> ~ `react-table`
2. <imports from common shared directories> ~ `common/types/types` | `helpers/helpers`
3. <imports from local directories/files> ~ `../helpers/helpers` | `./common/enums/enums`
4. <style import>
)
and don't forget about imports from same places - do not duplicate them
```suggestion
import { Column, useTable } from 'react-table';

import { FC } from 'common/types/types';
import { ReactTableRenderEntitiesNames } from './common/react-table-render-entities-names';

import styles from './styles.module.scss';
```